access.tex

```diff
@@ -0,0 +1,37 @@
+\section{Data Access Environment} \label{access}
+
+The Rubin data access environment provides data rights holders with access to all Rubin data products and services. 
+Prior to the start of Operations, all services for data access will be under active development and are provided on a shared-risk basis. 
+The Rubin data rights policy is described in  \citeds{RDO-013}.
+
+\subsection{Data Access Centers}
+
+Rubin data products will be served to the community from the US Data Access Center (US DAC) hosted on the Google Cloud Platform\footnote{\url{data.lsst.cloud}}.
+\hyperref[sec:dp01]{DP0.1} and \hyperref[sec:dp02]{DP0.2}  are already available via the RSP running at the US DAC.
+
+An number of Rubin Independent Data Access Centers (IDACs) is also under construction to provide additional user computing resources to LSST users around the globe (\citeds{RTN-003}).
+
+\subsection{Rubin Science Platform}
+
+The Rubin Science Platform (RSP), described in \citeds{LSE-319}, is a set of integrated web-based applications, services, and tools to query, visualize, subset, and analyze LSST data.
+It comprises three different ``Aspects'': a \emph{Portal} Aspect designed to provide an environment for data discovery, query, filtering, and visualization, a \emph{Notebook} Aspect to enable next-to-the-data analysis, and an \emph{API} Aspect that enables programmatic access to the Rubin data products via Virtual Observatory (VO) interfaces.
+The Portal and Notebook Aspects of the RSP make use of the same APIs as the API Aspect to internally access the LSST datasets.
+
+An early version of the RSP is currently running at the US Data Access Center (US DAC) that provides access to provides DP0.1 and DP0.2. 
+
+\subsubsection{Image services}
```

Comment:
I was expecting this subsection to mention the cutout / postage stamp service, and it's all-important connection to the notebook aspect as well as the API aspect. Maybe this belongs in the RSP subsection instead? Being able to obtain postage stamp images will be critical functionality for many science cases.

Reply:
It does mention cutouts: " an IVOA SODA service for making image cutouts" . It, and all image services are available as part of all Aspects. I could add a line making that clear

Reply:
Added line "Image services are accessible via the \emph{Portal}, \emph{Notebook}  and  \emph{API} aspects of the RSP."